wmfdata/hive.py

```diff
@@ -31,30 +51,12 @@ def run_cli(
         * "raw": a TSV string, as returned by the command line interface.
     * `heap_size`: the amount of memory available to the Hive client. Increase
       this if a command experiences an out of memory error.
-    * `use_nice`: Run with a lower priority for processor usage.
-    * `use_ionice`: Run with a lower priority for disk access.
     """
 
-    commands = ensure_list(commands)
     if format not in ["pandas", "raw"]:
-        raise ValueError("'{}' is not a valid format.".format(format))
-    if format == "raw":
-        warnings.warn(
-            "The 'raw' format is deprecated. It will be removed in the next major release.",
-            category=FutureWarning
-        )
-    check_kerberos_auth()
```

Comment:
Make sure to use this check in the `run` function.

Reply:
it doesn't seem to be necessary, pyhive seems to check it and print a proper error, do you think it's still useful?

Reply:
Well, the error we throw is much clearer and more helpful to the user:
> OSError: You do not have Kerberos credentials. Authenticate using `kinit` or run your script as a keytab-enabled user.

Compare that to the default error:
> TTransportException: Could not start SASL: b'Error in sasl_client_start (-1) SASL(-1): generic failure: GSSAPI Error: No credentials were supplied, or the credentials were unavailable or inaccessible (No Kerberos credentials available (default cache: FILE:/tmp/krb5cc_12049))'

Besides that, we use this function with all the run functions, so keeping it makes things consistent and gives us a single function we can modify later if the Kerberos setup changes.